### DISCUSSION TITLE
Could not load events

### DISCUSSION THREAD

**Discussion by bjendrick-bethany** [datalust/seq-tickets#1347] [Help]
Starting late last week, some of our signals and queries stopped loading and are now displaying the following error:
Could not load events
An unhandled error occurred while serving the request (token: 188c0e1d0a974e7c8acf7c5a578a751d).
We are on the latest verion of SEQ 6.0.6459.0. Ram cache coverage was at 0 days, we added more RAM to the VM Seq is running on at the moment RAM cache coverage is at 0.292 days not sure if that as anything to do with errors we are getting.

**Comment by nblumhardt** [comment]
Hello! I think we're on the matching support email thread for this - if you haven't received my reply over there, please just let me know :-)
